packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -817,6 +849,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     _cursorBlinkOpacityController.addListener(_onCursorColorTick);
     _floatingCursorResetController = AnimationController(vsync: this);
     _floatingCursorResetController.addListener(_onFloatingCursorResetTick);
+    _cursorVisibilityNotifier = ValueNotifier<bool>(widget.showCursor);
```

Comment:
This can then just be `_cursorVisibilityNotifier.value = widget.showCursor`.

Reply:
fixed